package.json

```diff
@@ -47,6 +48,13 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
     "prebuild": "prebuildify -t 8.14.0 -t electron@3.0.0 --napi --strip",
+    "prebuild:cross:android-arm": "prebuildify-cross --target android-arm",
+    "prebuild:cross:android-arm64": "prebuildify-cross --target android-arm64",
```

Comment:
@ahdinosaur So I guess these scripts is the main entry point for building and needs to be started somewhere, e.g. from `.travis.yml`. And we should run one Travis job for each one of them we'd like to build for, with different values for `--target`.

Reply:
Basically, since each of these scripts have a similar structure we could also do something like:

```json
"script": {
  "prebuildify-cross": "prebuildify-cross --target $TARGET"
}
```

And then set `$TARGET` from within `.travis.yml`. This makes it more generic and the `package.json` can settle in shape. Instead it's up to the caller of `npm run prebuildify-cross` to set the environment variable.